src/pages/index.js

```diff
@@ -4,6 +4,7 @@ import { Link } from "gatsby"
 import { Flex, Box, jsx } from "theme-ui"
 import { Spring, config } from "react-spring/renderprops"
 import useSound from "use-sound"
+import { Helmet } from "react-helmet"
```

Comment:
You must also use it like so `<Helmet title="🍾 🤣 🍾" />` or fetch the title from site metadata using graphQL, but for this site for now I would say it is ok to use Helmet directly with a static title.